template-transformer/handler.lua

```diff
@@ -144,8 +144,9 @@ function TemplateTransformerHandler:body_filter(config)
     end
 
     local headers = res_get_headers()
+    local internal = headers['Internal']
     local content_type = headers['Content-Type']
-    if content_type == "application/problem+json" then
+    if content_type == "application/problem+json" && internal == 1 then
```

Comment:
I think it should be a string, not a number

Reply:
done, thanks